code/datums/components/haunted.dm

```diff
@@ -0,0 +1,19 @@
+/datum/component/haunted
+
+/datum/component/haunted/Initialize(_strength=0, _source, _half_life=RAD_HALF_LIFE, _can_contaminate=TRUE)
```

Comment:
What's with all these arguments if you're not using any of them?

Reply:
copy pasta idiocracy from me, sorry.